core/src/main/java/tachyon/Constants.java

```diff
@@ -64,4 +67,9 @@
    * Version 3 [0.6.0] Add lastModificationTimeMs to inode.
    */
   public static final int JOURNAL_VERSION = 2;
+
+  public static final Class<? extends DataServer> DEFAULT_DATA_SERVER_CLASS =
```

Comment:
I also think it's fine to put it here, the reconstruction of conf will modify this.